src/main/java/seedu/address/commons/core/Config.java

```diff
@@ -17,6 +17,8 @@ public class Config {
     // Config values customizable through config file
     private Level logLevel = Level.INFO;
     private Path userPrefsFilePath = Paths.get("preferences.json");
+    private Path ingredientCatalogueFilePath = Paths.get("data/ingredientCatalogue.json");
```

Comment:
think these filepaths should be specified under the userPrefs class just like addressBookFilePath. Don't think you have to edit the config class